packages/config/config/devices/0x039a/doorWindow.json

```diff
@@ -0,0 +1,91 @@
+{
+	"manufacturer": "Inteset",
+	"manufacturerId": "0x039a",
+	"label": "Inteset Door/Window Sensor",
+	"description": "Door/Window Sensor",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0101"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Basic Set off Delay Time",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "2",
+			"label": "Basic Set Level",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100
```

Comment:
1. This isn't %. Basic Set commands go from 0 to 99.
2. The linked manual states that 100 maps to an "ON" command (0xff). This should be represented as an option

Reply:
Done.

Reply:
Looks like the checker isn't happy having options in addition to manual entry.   I don't think this is a partial parameter though rather the value it passes to the second association group?

Reply:
That's not the issue. You can have an option with manual entry.

Reply:
Gotcha!  I will correct.